### PR TITLE
Edit page

### PR DESCRIPTION
**Pull request by Ziad-Tari**
I created a profile edit page so the user can edit and add the required information.

### CODE REVIEW

**Review by ramy-att — CHANGES_REQUESTED** [review]
We need to add the student and employer options. @Ziad-Tari let me know when you are done and need help with the CSS, please.

**Comment by ramy-att** [comment]
@mario-elshaer @khaled106 added you as reviewers if you want to check the page and code! Thanks

File: client/package.json [hunk 1]
```diff
@@ -11,8 +12,9 @@
     "react-bootstrap": "^2.7.0",
     "react-bootstrap-icons": "^1.10.2",
     "react-dom": "^18.2.0",
+    "react-icons": "^4.7.1",
     "react-router-dom": "^5.3.4",
-    "react-scripts": "^2.1.3",
+    "react-scripts": "^5.0.1",
```
Comment (khaled106) [inline client/package.json:17]: Keep the one from Master. Pull from master and commit the changes they suggested

File: client/src/App.js [hunk 2]
```diff
@@ -1,10 +1,7 @@
 import React from "react";
-import { Route } from "react-router-dom";
```
Comment (khaled106) [inline client/src/App.js:2]: You need to pull from master first before making a Pull Request

File: client/src/App.js [hunk 3]
```diff
@@ -1,10 +1,7 @@
...
-import HomePage from "./pages/HomePage";
-import SignIn from "./pages/SignIn";
 import OurNav from "./components/NavBar/OurNav";
 import Footer from "./components/Footer/Footer";
-
+import Editprofile from "./pages/EditProfile";
```
Comment (khaled106) [inline client/src/App.js:4]: Make sure that all the imports in the master are here as well as the one you added!

**Review by khaled106 — CHANGES_REQUESTED** [review]
Make sure that you pull from origin first as some code are missing and this will cause a conflict

**Comment by ramy-att** [comment]
> Make sure that you pull from origin first as some code are missing and this will cause a conflict

@Ziad-Tari Please fix the pages and components in their correct files, pull the latest version of the project and push again with the issues resolved
@khaled106 thanks for your reviewing!

File: client/src/components/EditPage/EditPage.js [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+              className="resume"
+              controlId="exampleForm.ControlTextarea1"
+            >
+              <Form.Label>Resume</Form.Label>
+              <Form.Control
+                type="file"
+                value={file}
+                onChange={handlFileChange}
+              />
+            </Form.Group>
```
Comment (ramy-att) [inline client/src/components/EditPage/EditPage.js:157]: We should allow the user to have a default transcript too

**Review by ramy-att — CHANGES_REQUESTED** [review]

**Comment by ramy-att** [comment]
@Ziad-Tari It would make sense for us to complete #42 in this branch as well before merging. Please fix all the prior issues in the pr and try to complete #42. Let me or @khaled106 or @mario-elshaer know if you have any problems!

**Review by khaled106 — APPROVED** [review]
Good Work!! Thanks Ziad!!

**Review by ramy-att — APPROVED** [review]

**Pull request merged**